landscape.yml

```
@@ -51,6 +51,13 @@ landscape:
             logo: run-ai.svg
             twitter: https://twitter.com/runailabs
             crunchbase: https://www.crunchbase.com/organization/run-ai
+          - item:
+            name: Slinky
+            homepage_url: https://slinky.ai/
+            repo_url: https://github.com/SlinkyProject
```

The repo_url should be a valid GitHub repository url (https://github.com/cncf/landscape2/blob/58b7b22d5d36e1091d3e99e68b11467feaaa9197/docs/config/data.yml#L69)
```suggestion
            repo_url: null
```